tests/Byteflag.spec.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
```

Comment:
I figured it would be something like this, but I resent the fact that it's a string literal. =)